go.mod

```diff
@@ -1,19 +1,23 @@
 module github.com/e-conomic/go-imagecoding
 
-go 1.15
+go 1.20
 
 require (
 	github.com/Nr90/imgsim v0.0.0-20180202144352-5caa057144b0
 	github.com/disintegration/imaging v1.6.2
-	github.com/h2non/filetype v1.1.0
-	github.com/harukasan/go-libwebp v0.0.0-20190703060927-68562c9c99af
-	github.com/nfnt/resize v0.0.0-20180221191011-83c6a9932646 // indirect
-	github.com/pkg/errors v0.8.1 // indirect
+	github.com/h2non/filetype v1.1.3
+	github.com/harukasan/go-libwebp v0.0.0-20220408054828-61eedf90d768
 	github.com/rwcarlsen/goexif v0.0.0-20190401172101-9e8deecbddbd
-	github.com/stretchr/testify v1.4.0
-	github.com/strukturag/libheif v1.9.1
-	go.uber.org/atomic v1.4.0 // indirect
-	go.uber.org/multierr v1.1.0 // indirect
-	go.uber.org/zap v1.10.0
-	golang.org/x/image v0.0.0-20191009234506-e7c1f5e7dbb8
+	github.com/stretchr/testify v1.8.4
+	github.com/strukturag/libheif v1.11.0
```

Comment:
Can't really go higher then 1.11 because for whatever reason there are some missing bindings... I will leave it at this version for now and revisit later on